src/components/Header/Header.module.scss

```diff
@@ -1 +1,42 @@
+.wrapper {
+  border-bottom: 2px solid var(--color-neutral-400);
+}
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 14px 30px;
+}
 
+.logo {
+  font-size: 1.75rem;
+  font-family: var(--font-title);
+  font-weight: 800;
+}
+.links {
+  display: none;
+  text-transform: capitalize;
```

Comment:
Remove

Reply:
Hi, I tried to make this change but ended up keeping the display as none to remove them on smaller devices. Is there a better method you were thinking of?

Reply:
That's okay just meant to remove the text-transform